### PR TITLE
Issues 7

### PR DESCRIPTION
**Pull request by puranchand**

### CODE REVIEW

File: src/components/ElementList.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+ElementList.defaultProps = {
+    elements:[{
+        component:'',
+        props: {},
+        position:{
+            x:0,y:0
+        }
+    }],
```
Comment (manojsinghnegiwd) [inline src/components/ElementList.js]: If you want to set defaultProps to an array just assign a simple empty array. For e.g
```js
elements: []
```

File: src/components/ElementList.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    onMovableActive:() => void(0),
+    onMovableDown:() => void(0)
```
Comment (manojsinghnegiwd) [inline src/components/ElementList.js]: Please remove void(0) make it simple. For e.g
```js
onMovableActive:() => {}
```

File: src/components/ElementList.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            }
+        )
+	)
+}
+
+ElementList.propTypes = {
+    elements:PropTypes.array.isRequired,
+    onMovableActive:PropTypes.func.isRequired,
+    onMovableDown:PropTypes.func.isRequired
+
```
Comment (manojsinghnegiwd) [inline src/components/ElementList.js]: Please remove this space

**Review by manojsinghnegiwd — CHANGES_REQUESTED** [review]
Please have a look at my comments

File: src/components/ElementList.js (hunk 1 above)
Comment (puranchand) [reply src/components/ElementList.js]: Okay I will do this.

File: src/components/ElementList.js (hunk 2 above)
Comment (puranchand) [reply src/components/ElementList.js]: Okay

File: src/components/ElementList.js (hunk 3 above)
Comment (puranchand) [reply src/components/ElementList.js]: Okay

**Review by puranchand — COMMENTED** [review]
I have remove extra code Please check

**Comment by puranchand** [comment]
Done Pr https://github.com/manojsinghnegiwd/quick-cover/pull/12#pullrequestreview-139138966

**Review by manojsinghnegiwd — APPROVED** [review]
Seems good to me 👍

**Pull request merged**